src/style.css

```diff
@@ -0,0 +1,113 @@
+/* html {
+  @font-face {
+    font-family: Avenir;
+    src: url(fonts/Avenir-Book.ttf);
+    font-style: normal;
+    font-weight: 300;
+  }
+
+  @font-face {
+    font-family: Avenir;
+    src: url(fonts/Avenir-Heavy.ttf);
+    font-style: normal;
+    font-weight: 800;
+  }
+
+  font-family: Avenir, sans-serif;
+} */
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.search-box {
+  position: relative;
+  margin-top: 20px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  width: 100%;
+  border-radius: 4px;
+}
+
+.search-box.big {
+  height: 70px;
+}
+
+.search-box.small {
+  height: 42px;
+}
+
+.input {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  /* font-family: inherit; */
+  font-family: Avenir, sans-serif;
```

Comment:
why not inherit (it's better in this case)?

Reply:
![image](https://user-images.githubusercontent.com/121955055/214117751-be018096-b03c-4e9f-bb34-b49313918e53.png) - screenshot from my laptop
![image](https://user-images.githubusercontent.com/121955055/214118044-dcd14793-e4c8-492f-a4e6-1d6554cb9cd4.png) - test

Thanks for suggesting adding the fonts via DevTool. Inserting the fonts to html & using font-family: inherit looks fine from my end test doesn't see the changes I apply. should I nevertheless deploy it?